lib/linguist/heuristics.yml

```diff
@@ -113,6 +113,12 @@ disambiguations:
     #  : dependency
     # path/file.ext1 : some/path/../file.ext2
     pattern: '([\/\\].*:\s+.*\s\\$|: \\$|^[ %]:|^[\w\s\/\\.]+\w+\.\w+\s*:\s+[\w\s\/\\.]+\w+\.\w+)'
+- extensions: ['.dsp']
+  rules:
+  - language: Microsoft Developer Studio Project
+    pattern: '#.*Microsoft Developer Studio.*'
```

Comment:
I think we can replace this with only `# Microsoft Developer Studio Generated Build File` to avoid the any backtracking.

Could you also add support for these files in [`lib/linguist/generated.rb`](https://github.com/github/linguist/blob/master/lib/linguist/generated.rb), since they appear to be generated?